shell/platform/android/io/flutter/embedding/engine/mutatorsstack/FlutterMutatorView.java

```diff
@@ -74,23 +74,41 @@ public static boolean childHasFocus(@Nullable View root) {
     return false;
   }
 
+  @Nullable private ViewTreeObserver.OnGlobalFocusChangeListener focusListener;
+
   /**
    * Adds a focus change listener that notifies when the current view or any of its descendant views
    * have received focus.
    *
-   * @param focusListener The focus listener.
+   * <p>If there's an active focus listener, it will first remove the current listener, and then add
+   * the new one.
```

Comment:
This sentence is normally contradictory to the definition of an `addFoo` method. Consider naming it something more precise like `setOnDescendantFocusChangeListener`

Reply:
Agreed. Done